src/components/FileTree/index.spec.tsx

```diff
@@ -1,39 +1,37 @@
-import { Version, VersionEntryType } from '../../reducers/versions';
+import { Version, createInternalVersionEntry } from '../../reducers/versions';
+import { fakeVersionEntry } from '../../test-helpers';
 
-import { buildFileTree } from '.';
+import { buildFileTree, DirectoryNode } from '.';
 
 describe(__filename, () => {
   describe('buildFileTree', () => {
     it('creates a root node', () => {
       const versionId = '1234';
       const entries: Version['entries'] = [];
 
-      const data = buildFileTree(versionId, entries);
+      const tree = buildFileTree(versionId, entries);
 
-      expect(data).toEqual({
+      expect(tree).toEqual({
         id: `root-${versionId}`,
         name: versionId,
         children: [],
       });
     });
 
     it('converts a non-directory entry to a file node', () => {
-      const versionId = 'some-name';
+      const versionId = '1234';
 
       const filename = 'some-filename';
       const entries = [
-        {
-          depth: 0,
-          type: VersionEntryType.text,
+        createInternalVersionEntry({
+          ...fakeVersionEntry,
           filename,
```

Comment:
Maybe include `directory: false` just to be explicit?

Reply:
good call